Catappult/appcoins-wallet-android
# [APPC-4648] Android push notifications

**What does this PR do?**

   Added push notifications.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

  Create a new wallet. Check the token on backend and send a notification to the that wallet. Remove wallet. send a new notification to the same token and validate that it was not received.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4648](https://aptoide.atlassian.net/browse/APPC-4648)

**Code Review Checklist**



[APPC-4648]: https://aptoide.atlassian.net/browse/APPC-4648?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
